bundle-workflow/src/test_workflow/test_args.py

```diff
@@ -85,3 +85,6 @@ def __init__(self):
         self.component = args.component
         self.keep = args.keep
         self.logging_level = args.logging_level
+
+
+TestArgs.__test__ = False  # type:ignore
```

Comment:
Fixes a warning, see https://adamj.eu/tech/2020/07/28/how-to-fix-a-pytest-collection-warning-about-web-tests-test-app-class/.